### PR TITLE
[ContactStructuralMechanicsApplication] After last update on AMGCL some tests fails, changing to Skyline

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_mixedulm_linear_solver.cpp [hunk 1]
```diff
@@ -662,7 +660,7 @@ namespace Kratos
                 for (std::size_t j = 0; j < system_size; ++j) {
                     if ((((i == 0 || i == 1 || i == 2) && ((j == system_size - 1) || (j == system_size - 2) || (j == system_size - 3))) || ((j == 0 || j==1 || j==2) && ((i == system_size - 1) || (i == system_size - 2) || (i == system_size - 3))) || (i == 6 && (j == 10 || j == 11)) || (i == 7 && (j == 9 || j == 11)) || (i == 8 && (j == 9 || j == 10))) == false) {
                         count += 1.0;
-                        A.push_back(i, j, std::sqrt(count));
+                        A.push_back(i, j, 5.0 + std::sqrt(count));
```
Comment (loumalouomega) [inline applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_mixedulm_linear_solver.cpp:656]: I do this to create a "random" system better conditioned (that's why I used AMGCL)

**Review by philbucher — DISMISSED** [review]
ok

**Comment by loumalouomega** [comment]
@philbucher Can you reapprove?, evil model created a conflict

**Review by philbucher — APPROVED** [review]

**Comment by philbucher** [comment]
9sec to reapprove, can anyone beat that? :D

**Comment by philbucher** [comment]
screenshot from 2018-10-18 10-08-52

**Comment by loumalouomega** [comment]
> screenshot from 2018-10-18 10-08-52

Probably Sanic

https://www.youtube.com/watch?v=paZcBwCLLBM

Gotta go fast

**Pull request merged**